source/docs/articles/sites/platform-considerations.md

```diff
@@ -41,6 +41,9 @@ Pantheon cannot be used to host files over 256MB, no matter how the file is upl
 
 If you are distributing large binaries or hosting big media files, we recommend using a CDN like Amazon S3 as a cost-effective file serving solution. Modules such as [Amazon S3 CORS Upload](https://drupal.org/project/amazons3_cors) allow uploads directly to S3 from your Drupal site without using Pantheon as an intermediary.
 
+## Large Code Repository
+When a code repo is larger than 2GB, it slows deploys to production and decreases functionality. We suggest keeping multimedia assets out of the repo by moving them to a media file storage service such as [Amazon S3](https://aws.amazon.com/s3/), and using version control to track URLs.
```

Comment:
@erik-pantheon can large repos also prevent the ability to commit OSD changes and prevent merges into Master?

Are there other kinds of files that might bloat the code repo to this kind of size?